inst/gates/master_table.tsv

```diff
@@ -20,3 +20,11 @@ Talphabeta	TRAC;TRBC1;TRBC2
 pDC	CCR9;LILRA4
 Erythrocyte	HBB;HBA2;HBA1
 Platelet_MK	PPBP;ITGB3
+Erythrocyte.RM	HBA1;TMSB10-;ACTG1-;ACTB-
```

Comment:
is HBB not actually a good erythrocyte marker?

Reply:
HBB expression seems to spread more into neighboring clusters than HBA1, so including it filters out more data.